python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```diff
@@ -461,6 +488,15 @@ $defs:
             description: IPv6 subnet for Loopback0 allocation.
             type: str
             format: ipv6_cidr
+          loopback_ipv6_prefix_length:
```

Comment:
As noted some other random place, please move this to fabric_ip_addressing together with other similar knobs.